mergify_engine/web/root.py

```diff
@@ -172,6 +173,20 @@ async def refresh_branch(
     return responses.Response("Refresh queued", status_code=202)
 
 
+@app.get(
+    "/organization/{owner_id}/usage",  # noqa: FS003
+    dependencies=[fastapi.Depends(auth.signature)],
+)
+async def get_stats(
+    owner_id: github_types.GitHubAccountIdType,
+    redis_cache: utils.RedisCache = fastapi.Depends(  # noqa: B008
+        redis.get_redis_cache
+    ),
+) -> responses.Response:
+    seats = (await count_seats.Seats.get(redis_cache, write=False)).count()
+    return responses.JSONResponse(content={"active_users": seats.active})
```

Comment:
funny you used `active_users` here and I do think it's better indeed.

I'd rather use `seats.jsonify()` here WDYT?

Reply:
I will make a better json then